libs/server-sdk/include/launchdarkly/server_side/client.hpp

```diff
@@ -0,0 +1,331 @@
+#pragma once
+
+#include <launchdarkly/config/server.hpp>
+#include <launchdarkly/context.hpp>
+#include <launchdarkly/data/evaluation_detail.hpp>
+#include <launchdarkly/value.hpp>
+
+#include <chrono>
+#include <future>
+#include <memory>
+#include <string>
+#include <unordered_map>
+
+namespace launchdarkly::server_side {
+
+/**
+ *  Interface for the standard SDK client methods and properties.
+ */
+class IClient {
```

Comment:
This whole interface is copied from the client, so adjustments will need to be made in subsequent PRs.